docs/code-search/code-navigation/envvars.mdx

```diff
@@ -1,12 +1,12 @@
-# Code navigation environment variables
+# Code Navigation Environment Variables
 
-All of the following environment variables are prefixed with `PRECISE_CODE_INTEL`.
+<p className="subtitle">These docs list supported environment variables for Code Navigation. All of these environment variables are prefixed with `PRECISE_CODE_INTEL`.</p>
 
 ## frontend
 
 The following are variables are read from the `frontend` service to control code navigation behavior exposed via the GraphQL API.
 
-|                       Name                        | Default |                          Description                          |
+|                       **Name**                        | **Default** |                          **Description**                          |
```

Comment:
It seems like all tables have bold headings. Is it possible to move the styling to the Markdown processing layer to avoid cluttering headings with extra `**` marks for each header column?